### PR TITLE
Amend release process description

### PR DESCRIPTION
**Pull request by jtama**

### CODE REVIEW

File: docs/modules/project/pages/hacking.adoc [hunk 1]
```diff
@@ -368,25 +368,33 @@ In addition, you can submit pull requests to downstream project:
 
 [%interactive]
 * [ ] submit a pull request downstream to update Asciidoctor reveal.js version in the Asciidoctor Docker Container
-** modify the `Dockerfile`, `Makefile` and `README.adoc` in: https://github.com/asciidoctor/docker-asciidoctor
+** modify the `Dockerfile`, `tests/asciidoctor.bats`, `README.md` and `README.adoc` in: https://github.com/asciidoctor/docker-asciidoctor
```
Comment (ggrossetie) [inline docs/modules/project/pages/hacking.adoc:371]: I feel like this should be documented at https://github.com/asciidoctor/docker-asciidoctor "How to bump a version". What do you think @dduportal?

File: docs/modules/project/pages/hacking.adoc [hunk 2]
```diff
@@ -368,25 +368,33 @@ In addition, you can submit pull requests to downstream project:
...
 * [ ] submit a pull request downstream to update AsciidoctorJ reveal.js version
-** modify `gradle.properties`, `asciidoctorj-revealjs/gradle.properties` and `asciidoctorj-revealjs/build.gradle` in: https://github.com/asciidoctor/asciidoctorj-reveal.js
+** modify `gradle.properties` and `asciidoctorj-revealjs/gradle.properties`. if `reveal.js`version has been updated, +
+also ammend `WhenBackendIsRevealJs.java` in both `tests` and `itests` folders and `README.adoc` in: https://github.com/asciidoctor/asciidoctorj-reveal.js
```
Comment (ggrossetie) [inline docs/modules/project/pages/hacking.adoc:374]:
```suggestion
also amend `WhenBackendIsRevealJs.java` in both `tests` and `itests` folders and `README.adoc` in: https://github.com/asciidoctor/asciidoctorj-reveal.js
```

File: docs/modules/project/pages/hacking.adoc (hunk 1 above)
Comment (jtama) [reply docs/modules/project/pages/hacking.adoc:371]: Then we could simply link the related documentation.

Comment (ggrossetie) [reply docs/modules/project/pages/hacking.adoc:371]: We can move forward while waiting for Damien input. Feel free to merge.

**Review by ggrossetie — APPROVED** [review]
LGTM

File: docs/modules/project/pages/hacking.adoc (hunk 1 above)
Comment (dduportal) [reply docs/modules/project/pages/hacking.adoc:371]: Hello folks! That is, indeed, a good idea. Let me start a PR in the docker image PR for documenting it, and i'll link it in this doc afterwards.

Comment (dduportal) [reply docs/modules/project/pages/hacking.adoc:371]: I've opened https://github.com/asciidoctor/docker-asciidoctor/pull/486 on the image side. May I have your review on it?

I'll open a PR here once merged on image side

**Pull request merged**